src/Types.ts

```diff
@@ -125,8 +125,11 @@ export type PrimitiveFunctionName =
   'requireBlockNotMined' |
   'requireBlockMined' |
   'requireUint256LowerBound' |
+  'requireUint256UpperBound' |
   'limitSwapExactInput' |
-  'blockInterval'
+  'blockInterval' | 
+  'requireBitNotUsed' |
```

Comment:
I added these so you'll have to rebase and resolve the conflict

Reply:
@phanpak need to fix the conflict with base here still

Reply:
sorry i didn't get it. done!